aerogear/mobile-developer-console
# Error notifications

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8047
https://issues.jboss.org/browse/AEROGEAR-7964

## Verification Steps
Verify that there are correct error messages shown in UI.

1. `./scripts/development.sh`
2. Try to create app with name that already exist
3. Open url for app name that does not exist
4. Try to kill backend with `lsof -i tcp:4000 | grep LISTEN | awk '{print $2}' | xargs kill`
5. restart MDC
6. `oc logout`

## Checklist:
